src/components/Dropdown.jsx

```diff
@@ -30,5 +31,12 @@ class Dropdown extends React.Component {
 		);
 	}
 }
+Dropdown.propTypes = {
+	options: PropTypes.shape({
+		name: PropTypes.string.isRequired,
+		value: PropTypes.string.isRequired,
+		onChange: PropTypes.func.isRequired,
+	}),
```

Comment:
So name, value, and onChange are all separate from options. You don't access them like
```
this.props.options.name
```
they're just
```
this.props.name
```
etc.